scripts/quests/otherAreas/Elderly_Pursuits.lua

```diff
@@ -0,0 +1,158 @@
+-----------------------------------
+-- Elderly Pursuits
+-----------------------------------
+-- Log ID: 4, Quest ID: 75
+-----------------------------------
+-- Pre-requisite Completed Secrets of Oven Lost Log ID: 4, Quest ID: 73
+-----------------------------------
+-- Despachiaire !gotoid 16883721
+-- Rouva !gotoid 17719492
+-- Para NM ??? !gotoid 16785771
+-----------------------------------
```

Comment:
This is a nitpick, however, entity IDs can and will shift at any moment on a version update. Could this be updated with their positions with the use of !pos, similar to other quests?

Reply:
Done.